code/modules/clothing/spacesuits/void.dm

```diff
@@ -1,14 +1,95 @@
 
-//NASA Voidsuit
-/obj/item/clothing/head/helmet/space/nasavoid
-	name = "NASA Void Helmet"

```

Comment:
This is the future though. _Everything_ should be retro at this point. Also, ewwwww retro

Reply:
Not in favour of that. They're named NASA voidsuits as a verbatim and intentional reference.

Reply:
Well, I would be happy to change them back if people want me to

Reply:
I just ask one thing, should I have the color in the name, or just have it be NASA Void Helmet/Suit?

Reply:
I'd vote have the color in the description rather in the name. Also if using colors would it be possible to streamline the objects? You said they would just spawn in so why not just make one instance and have them spawn via color pick?

Reply:
Hmm, that is a good idea, but the problem with that is that if it's on both the suit and helmet it may be nearly impossible to have matching suits and helmets, as well as being impossible to pick your suit color.

Reply:
You mean the spawner can't spawn both the suit and helmet together? Also random doesn't normally mean pick, unless for admeme purposes you can pass an optional var into the spawn proc. Not sure if that's possible

Reply:
Oh, yes, I was going to make either a proc or an item that just randomly picks a suit and helmet, spawns them, and if it was an item, delete the item.